ui/src/Elements/Message/Message.assets.js

```diff
@@ -0,0 +1,12 @@
+export const CONSUMER = 'consumer';
+export const PRODUCER = 'producer';
+
+export const translations = {
```

Comment:
Could the translations be put into a seperate i18n file, as per https://github.com/matthew-chirgwin/kafka-java-vertx-starter/tree/ui/ui/src/Elements#file-structure ? 

They then can be accessed/used via assets by doing `export * from 'Message.i18n.json'`

My thought/aim here is to have an easy to find file suffix, which is then available for test/storybook/product usage - hence this particulalr structure.

Reply:
Got it - have updated.